# Time of status message
time t

# Current position, in degrees
float64 curr_lat
float64 curr_lon

# Destination lat lon
float64 dest_lat
float64 dest_lon

# Status 0: From the model, 1: random.
uint8 status
